ui/app/pages/settings/networks-tab/network-form/network-form.component.js

```diff
@@ -24,6 +24,7 @@ export default class NetworkForm extends PureComponent {
     isCurrentRpcTarget: PropTypes.bool,
     blockExplorerUrl: PropTypes.string,
     rpcPrefs: PropTypes.object,
+    networksToRender: PropTypes.array,
```

Comment:
We'd get a better type guarantee here if we used `PropTypes.arrayOf(PropTypes.object)`, as any other element would cause the render to crash.
Or we could move the `networksToRender.map(...)` step to the `network-tab` component, and only pass down the array of URLs.